common/lib/container-definitions/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@fluidframework/common-definitions": "^0.20.0",
-    "@fluidframework/core-interfaces": "^0.39.0",
+    "@fluidframework/core-interfaces": "^0.39.5",
```

Comment:
Not sure if this needs to be reverted to `"^0.39.0"`. I guess `"^0.39.5"` is more explicit so I kept it as is.